core/templates/services/context.service.ts

```diff
@@ -132,39 +130,6 @@ export class ContextService {
     return this.questionPlayerIsManuallySet;
   }
 
-  setContributorDashboardQuestionEditorIsOpen(): void {
-    this.contributorDashboardQuestionEditorIsOpen = true;
-  }
-
-  clearContributorDashboardQuestionEditorIsOpen(): void {
-    this.contributorDashboardQuestionEditorIsOpen = false;
-  }
-
-  getContributorDashboardQuestionEditorIsOpen(): boolean {
-    return this.contributorDashboardQuestionEditorIsOpen;
-  }
-
-  canEntityReferToSkills(): boolean {
-    return (
-      this.getPageContext() === ServicesConstants.PAGE_CONTEXT.TOPIC_EDITOR ||
-      this.getPageContext() === ServicesConstants.PAGE_CONTEXT.SKILL_EDITOR ||
-      (
-        this.getPageContext() === (
-          ServicesConstants.PAGE_CONTEXT.CONTRIBUTOR_DASHBOARD) &&
-          this.getContributorDashboardQuestionEditorIsOpen()
-      ) ||
-      (
-        this.getPageContext() === (
-          ServicesConstants.PAGE_CONTEXT.EXPLORATION_EDITOR) &&
-        this.explorationIsLinkedToStory
-      )
-    );
-  }
-
-  setExplorationIsLinkedToStory(): void {
-    this.explorationIsLinkedToStory = true;
-  }
-
```

Comment:
Similar comment as last time, can you not remove this, since this is going to be used :P.

Reply:
Done.